src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModelFactory.cs

```diff
@@ -118,11 +92,32 @@ public PackageModel Create(PackageSearchMetadataContextInfo metadata, ContractIt
             }
             else
             {
-                PackageMetadataRetrievalAdapter packageMetadataRetrievalAdapter = new PackageMetadataRetrievalAdapter(_searchService, metadata.Identity!, _packageSources, _includePrerelease);
-                IDeprecationCapable deprecationCapable = new DeprecationPackageMetadataCapability(packageMetadataRetrievalAdapter);
-                VulnerablePackageMetadataCapability vulnerableCapability = new VulnerablePackageMetadataCapability(packageMetadataRetrievalAdapter);
+                // Transitive dependencies are only available in the Installed tab
+                if (metadata.TransitiveOrigins != null && itemFilter.Equals(ContractItemFilter.Installed))
+                {
+                    IVulnerableCapable vulnerableDatabaseCapability = new VulnerableDatabaseCapability(_packageVulnerabilityService, metadata.Identity);
+                    return new TransitivelyReferencedPackageModel(
+                        metadata.Identity!,
+                        vulnerableDatabaseCapability,
+                        embeddedResources,
+                        metadata.TransitiveOrigins,
+                        metadata.Title,
+                        metadata.Description,
+                        metadata.Authors,
+                        metadata.ProjectUrl,
+                        metadata.Tags?.Split(','),
+                        metadata.OwnersList,
+                        metadata.DependencySets,
+                        metadata.Summary,
+                        metadata.Published,
+                        metadata.LicenseMetadata,
+                        metadata.LicenseUrl,
+                        metadata.RequireLicenseAcceptance,
+                        metadata.ReportAbuseUrl?.ToString(),
+                        metadata.IconUrl);
+                }
 
-                if (metadata.IsRecommended)
+                if (metadata.IsRecommended && itemFilter.Equals(ContractItemFilter.All))
```

Comment:
Same as transitive, this property is enough to indicate that the package is Recommended.

```suggestion
                if (metadata.IsRecommended)
```